src/Raven.Client/Constants.cs

```diff
@@ -220,6 +220,8 @@ private Querying()
                 }
 
                 public const string IntersectSeparator = " INTERSECT ";
+
+                public const string DynamicQueryPrefix = "dynamic/";
```

Comment:
I do not like having mutually exclusive parameters... but I do not see a better solution for the moment.